ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12__amd64-nightly-4.12-upgrade-from-stable-4.12.yaml

```diff
@@ -238,6 +238,18 @@ tests:
     - ref: cucushift-installer-check
     - chain: openshift-upgrade-qe-test
     workflow: cucushift-installer-rehearse-aws-ipi-proxy-cco-manual-security-token-service
+- as: aws-ipi-rm-root-secret-f14
+  cron: 47 13 5,20 * *
```

Comment:
shall we add one more destructive testing? and arm jobs?

Reply:
The intention of this PR is to focus on updates, do you mean if we cover `rm-root-secret` in general e2e workflow?

Reply:
> do you mean if we cover rm-root-secret in general e2e workflow?
@jianlinliu I think we can pick 1-2 workflows to cover this config. WDYT?

Reply:
> The intention of this PR is to focus on updates

Ah, forgot that. Ignore the comment.

Reply:
> Ignore the comment.

I meant we can ignore  destructive testing comments. But maybe we can add one more ci job for arm upgrade case?

Reply:
Updated.